app/config/locale/translations/la.json

```diff
@@ -0,0 +1,232 @@
+{
+  "settings.inspire": "\"El Arte de ser sabio es el arte de saber qué pasar por alto.\"",
+  "settings.locale": "en-la",
+  "settings.direction": "ltr",
+  "emails.sender": "%s equipo",
+  "emails.verification.subject": "Verificación de cuenta",
+  "emails.verification.hello": "Hola {{name}}",
+  "emails.verification.body": "Sigue este vínculo para verificar tu correo electrónico.",
+  "emails.verification.footer": "Si no solicitaste verificación de este correo electrónico, puedes ignorar este mensaje.",
+  "emails.verification.thanks": "Gracias",
+  "emails.verification.signature": "{{project}} Equipo",
+  "emails.magicSession.subject": "Iniciar sesion",
+  "emails.magicSession.hello": "Hola,",
+  "emails.magicSession.body": "Sigue este vínculo para iniciar sesion",
+  "emails.magicSession.footer": "Si no solicitaste verificación de este correo electrónico, puedes ignorar este mensaje.",
+  "emails.magicSession.thanks": "Gracias",
+  "emails.magicSession.signature": "{{project}} equipo",
+  "emails.recovery.subject": "Recuperar contraseña",
+  "emails.recovery.hello": "Hola {{name}}",
+  "emails.recovery.body": "Sigue este vínculo para recuperar la contraseña de tu proyecto {{project}}",
+  "emails.recovery.footer": "Si no solicitaste recuperar tu contraseña, Puedes ignorar este mensaje",
+  "emails.recovery.thanks": "Gracias",
+  "emails.recovery.signature": "{{project}} equipo",
+  "emails.invitation.subject": "Invitacion para %s en el equipo %s",
+  "emails.invitation.hello": "Hola",
+  "emails.invitation.body": "Este correo electrónico te fue enviado por que el dueño del proyecto quiere invitarte a convertirte en miembro en el equipo {{team}} en el {{project}}",
```

Comment:
https://github.com/appwrite/appwrite/blob/master/app/config/locale/translations/en.json#L26

You're missing the `{{owner}}` placeholder